install.sh

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
+
+set -u
+
+DOTENV_LINTER="dotenv-linter"
+DOTENV_LINTER_REPO="${DOTENV_LINTER}/${DOTENV_LINTER}"
+DOTENV_LINTER_GITHUB="https://github.com/${DOTENV_LINTER_REPO}"
+DOTENV_LINTER_RELEASE_API="https://api.github.com/repos/${DOTENV_LINTER_REPO}/releases/latest"
+
+main() {
+    require_cmd uname
+    require_cmd mkdir
+    require_cmd grep
+    require_cmd cut
+    require_cmd rm
+
+    local _target_version
+    if [ $# -eq 0 ]; then
+        get_latest_version || return 1
+        _target_version="$RETVAL"
+    else
+        _target_version="$1"
+    fi
+
+    get_architecture || return 1
+    local _arch="$RETVAL"
+
+    local _ext
+    case $_arch in
+        win-*) _ext=".zip" ;;
+        *) _ext=".tar.gz" ;;
+    esac
+
+    local _archive_name="${DOTENV_LINTER}-${_arch}${_ext}"
+    local _url="${DOTENV_LINTER_GITHUB}/releases/download/${_target_version}/${_archive_name}"
+
+    # Installation
+    local _dir="$HOME/.${DOTENV_LINTER}"
```

Comment:
I'd like to install dotenv-linter to the `bin` directory in the current directory.
Mostly, this script will be used in CI-services. So, I wouldn't like to install the project globally.

I've seen a lot of projects with the similar approach. For example:
https://github.com/reviewdog/reviewdog#installation
https://github.com/reviewdog/reviewdog/blob/master/install.sh#L12

Reply:
Actually, I agree.
I posted this version to consult exactly on the installation method.

By and large, I am inspired by [rustup](sh.rustup.rs) _sh installer_. It has a similar way.
I also looked at the reviewdog installer, it looks good, and I also wanted to use a special flag for `bindir` and will do it soon, since I like the approach without changing the environment variables more